sklearn/ensemble/_bagging.py

```diff
@@ -936,6 +997,14 @@ def predict_proba(self, X):
             reset=False,
         )
 
+        _raise_for_params(params, self, "predict_proba")
+
+        if _routing_enabled():
+            routed_params = process_routing(self, "predict_proba", **params)
+        else:
+            routed_params = Bunch()
+            routed_params.estimator = Bunch(predict_proba=params)
```

Comment:
we don't really support routing metadata around if metadata routing is disabled anyway (the `_raise_for_params` above). So here the `Bunch` can/should be empty.